ludwig/models/llm.py

```diff
@@ -135,9 +135,10 @@ def __init__(
         else:
             self.context_len = 2048
 
+        self.generation = GenerationConfig(**self.config_obj.generation.to_dict())
         # max input length value copied from FastChat
         # https://github.com/lm-sys/FastChat/blob/0e958b852a14f4bef5f0e9d7a5e7373477329cf2/fastchat/serve/inference.py#L183  # noqa
-        self.max_new_tokens = self.config_obj.generation.max_new_tokens
+        self.max_new_tokens = self.generation.max_new_tokens
```

Comment:
Good catch finding these other variables that also need to be updated when GenerationConfig is updated! OOC, where does 8 come from?

Reply:
from the comment above (FastChat implementation we grandfathered in): https://github.com/lm-sys/FastChat/blob/0e958b852a14f4bef5f0e9d7a5e7373477329cf2/fastchat/serve/inference.py#L183